pkgs/by-name/us/ustreamer/package.nix

```diff
@@ -36,6 +39,16 @@ stdenv.mkDerivation rec {
     libjpeg
     libdrm
   ]
+  ++ lib.optionals withPython (
+    with python3Packages;
+    [
+      python3
+      setuptools
+      wheel
+      build
+      pip
```

Comment:
You are sure about those 4 packages? Usually adding wheel is not necessary in all python packages. Also how is pip used?